src/modules/remove.module.js

```diff
@@ -0,0 +1,16 @@
+/* eslint "import/prefer-default-export":"off" */
+import { readLocalStorage, writeLocalStorage } from './local-storage.module.js';
+
+function removeItems(toDelete) {
+  const items = readLocalStorage();
+  toDelete.forEach((element) => {
+    items.splice(toDelete[element - 1], 1);
+  });
+  items.forEach((item, index) => {
+    item.id = index + 1;
+    items[index] = item;
+  });
+  writeLocalStorage(items);
+}
```

Comment:
- [ ] Let's make sure you are deleting the completed tasks(selected). For now when `Clear all completed` button is clicked only the first task is deleted.

Reply:
it clears all the items in the toDelete array, then I have to update id's of the remaining items as indicated in the requirements @uwadonat